### DISCUSSION TITLE
Feature request: support release details

### DISCUSSION THREAD

**Discussion by keith** [community/community#3] [Mobile]
Currently if you "watch releases" on a repository, and get a notification about them, tapping it opens Safari. It would be awesome if these were shown in the app instead!

**Comment by candyho** [comment]
Hey @keith , thanks for your feedback! We feel you and this is something on our backlog. We’ll continue to monitor related feedback and re-evaluate the priority regularly.

**Comment by TheGiraffe3** [comment]
Does this still happen?
@github, have any steps been taken?
